server/plugin.go

```diff
@@ -84,102 +83,96 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	var err error
 	path := r.URL.Path
 	switch path {
 	case "/api/v1/product_type":
-		p.handleProductType(w, r)
+		err = p.handleProductType(w, r)
 	case "/api/v1/create_meeting_in_server_version":
 		p.handleCreateMeetingInServerVersion(w, r)
```

Comment:
Why is `handleCreateMeetingInServerVersion` not returning an `error`?

Reply:
I am curious about this too. `handleCreateMeetingInServerVersion` has a different shape than the rest of the functions. I can see that it was like that before too (it was using mlog instead of print) but if you could make everything work the same way it would be better.

Reply:
Indeed, I forgot `handleCreateMeetingInServerVersion`. Done in the most recent commit :+1: